tests/wildcard.rs

```diff
@@ -1,34 +1,11 @@
 #![feature(futures_api, async_await)]
 
-use futures::{executor::block_on, prelude::*};
-use http_service::{Body, HttpService, Request, Response};
+use http_service::{Body};
+use futures::{executor::block_on};
 use tide::{
-    head::{Named, NamedSegment},
-    Server,
+    head::{Named, NamedSegment}
 };
-
-struct TestBackend<T: HttpService> {
-    service: T,
-    connection: T::Connection,
-}
-
-impl<T: HttpService> TestBackend<T> {
-    fn wrap(service: T) -> Result<Self, <T::ConnectionFuture as TryFuture>::Error> {
-        let connection = block_on(service.connect().into_future())?;
-        Ok(Self {
-            service,
-            connection,
-        })
-    }
-
-    fn simulate(&mut self, req: Request) -> Result<Response, <T::Fut as TryFuture>::Error> {
-        block_on(
-            self.service
-                .respond(&mut self.connection, req)
-                .into_future(),
-        )
-    }
-}
+use http_service_mock::{make_server};
```

Comment:
Since this is only being used in the test module, wouldn't moving it inside the tests module reduce overall binary size?

Reply:
Could you elaborate on this or expand your explanation? I don't seem to get it! As I see it, `http_servicemock` is already a `dev-dependency` and therefore won't be included in the build?

Reply:
My bad, I didn't see that it was as already a dev-dependency.